app/javascript/dashboard/components/widgets/conversation/Message.vue

```diff
@@ -274,6 +285,9 @@ export default {
     isTemplate() {
       return this.data.message_type === MESSAGE_TYPE.TEMPLATE;
     },
+    isAnIntegrationMessage() {
+      return this.contentType === 'integrations';
```

Comment:
contentType constants are not defined in the frontend code. We can pick it up as a chore.